backend/ticketvise/utils.py

```diff
@@ -3,12 +3,13 @@
 -------------------------------
 Various utility functions for use in python files.
 """
+import random
 
 from PIL import ImageColor
 
-from .models.ticket import Status
-from .models.user import Role
-from .settings import DEFAULT_AVATAR_PATH
+
+def random_color():
+    return "#%06x" % random.randint(0, 0xFFFFFF)
```

Comment:
Zorgt dit niet voor hele vreemde of vage kleuren? Beter misschien om een set van een aantal kleuren te geven waar willekeurig een van gepakt wordt of om het af te bakenen